src/code/z_scene.c

```diff
@@ -535,57 +535,58 @@ void Scene_SetExitFade(PlayState* play) {
     play->transitionType = Entrance_GetTransitionFlags(play->nextEntrance) & 0x7F;
 }
 
+static void (*sceneCmdHandlers[])(PlayState*, SceneCmd*) = {
+    Scene_HeaderCmdSpawnList,
```

Comment:
Could probably rename these with the header now as well.

Reply:
Took to the OoT style and went `Scene_HeaderCmd` -> `Scene_Command`
also
`sceneCmdHandlers` -> `sSceneCmdHandlers`

Reply:
just for clarity, I dislike this naming scheme and have proposed `CutsceneCmd_` as a new system name in z_demo docs, and think if that is agreed on, it can be used for other command handler functions as well in other scripting systems

Not saying to use this now, but just saying that I dont think this style in oot is 100% solidified

Reply:
Good to note, thanks. My preference is to just leave it as I've changed it for now, and just sync later once new names are decided on in OoT if we like them